deps/rabbit/src/rabbit_amqqueue.erl

```diff
@@ -1603,7 +1603,12 @@ delete_immediately_by_resource(Resources) ->
             rabbit_types:error('not_empty') |
             {protocol_error, Type :: atom(), Reason :: string(), Args :: term()}.
 delete(Q, IfUnused, IfEmpty, ActingUser) ->
-    rabbit_queue_type:delete(Q, IfUnused, IfEmpty, ActingUser).
+    rabbit_log:warning("TRACE delete queue ~p~n before ~p~n",
```

Comment:
Looks like this should be turned into a debug statement or dropped entirely

Reply:
Thanks, it completely slip through. It's gone now